test/src/concrete/vault/ERC20PriceOracleReceiptVault.mint.t.sol

```diff
@@ -158,5 +160,28 @@ contract ERC20PriceOracleReceiptVaultDepositTest is ERC20PriceOracleReceiptVault
         vm.stopPrank();
     }
 
+    /// forge-config: default.fuzz.runs = 1
+    function testMintFlareFork(uint256 amount) public {
+        amount = bound(amount, 1, type(uint128).max);
+
+        // Approve an amount slightly larger than `amount` to avoid allowance issues
+        (ERC20PriceOracleReceiptVault vault, address alice) =
```

Comment:
you shouldn't need to do this, can't you use `previewMint` to get the right amount?